### PR TITLE
New Admin APIs

### PR DESCRIPTION
**Pull request by timoha**
- Add `SetBalancer` to enable/disable balancer
- Add `MoveRegion` to move regions

### CODE REVIEW

File: hrpc/move.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+// WithDestinationRegionServer specifies destination RegionServer for MoveReqion request
+// A server name is its host, port plus startcode: host187.example.com,60020,1289493121758
+func WithDestinationRegionServer(serverName string) func(Call) error {
+	return func(c Call) error {
+		mr, ok := c.(*MoveRegion)
+		if !ok {
+			return errors.New("WithDestinationRegionServer option can only be used with MoveRegion")
+		}
+		out := strings.SplitN(serverName, ",", 3)
```
Comment (tsuna) [inline hrpc/move.go:33]: Need to check that `len(out)` is 3. `SplitN()` returns _at most_ N substrings, but could return 0 or 1 or 2.

**Review by tsuna — CHANGES_REQUESTED** [review]

File: hrpc/move.go (hunk 1 above)
Comment (timoha) [reply hrpc/move.go:33]: Done

File: hrpc/move.go [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+		if len(out) != 3 {
+			return errors.New("invalid server name, needs to be of format <host>,<port><startcode>")
```
Comment (tsuna) [inline hrpc/move.go:35]: Missing a comma after <port>

**Review by tsuna — CHANGES_REQUESTED** [review]

File: hrpc/move.go (hunk 2 above)
Comment (timoha) [reply hrpc/move.go]: Done

**Review by tsuna — APPROVED** [review]

**Pull request merged**